plugins/base/build/command_test.go

```diff
@@ -38,3 +43,42 @@ func TestSetEnv(t *testing.T) {
 
 	})
 }
+
+func TestCommand_Run(t *testing.T) {
```

Comment:
This is a good start. Could we add a test for another builder command (one built for testing) so that we can ensure this will work with anything we throw at it? 

There are some main goals we should make evident in testing:
- That it stops when one of the BeforeBuilders fails.
- That it does not execute the subsequent BeforeBuilder
- That it does not execute the Builders
- That it does not execute the AfterBuilders.